test/testcafe/spec/SelectAuthenticatorForVerification_spec.js

```diff
@@ -103,6 +103,20 @@ test.requestHooks(mockChallengePassword)(`should navigate to password challenge
   await t.expect(challengeFactorPage.getPageTitle()).eql('Sign in using your password');
 });
 
+test.requestHooks(mockChallengePassword)(`select password challenge page and hit switch authenticator and re-select password`, async t => {
+  const selectFactorPage = await setup(t);
+  await t.expect(selectFactorPage.getFormTitle()).eql('Verify it\'s you with an authenticator');
+
+  selectFactorPage.selectFactorByIndex(0);
+  const challengeFactorPage = new ChallengeFactorPageObject(t);
+  await t.expect(challengeFactorPage.getPageTitle()).eql('Sign in using your password');
+  await challengeFactorPage.clickSwitchAuthenticatorButton();
```

Comment:
after this, should we add a check to see if we are actually at the select factor page?